src/utils.jl

```diff
@@ -12,15 +12,35 @@ end
 #                           DifferentiationMethod
 ################################################################################
 
+supports_autoenzyme(::Type{<:Meshes.Geometry}) = true
+supports_autoenzyme(::Type{<:Meshes.BezierCurve}) = false
+supports_autoenzyme(::Type{<:Meshes.CylinderSurface}) = false
+supports_autoenzyme(::Type{<:Meshes.Cylinder}) = false
+supports_autoenzyme(::Type{<:Meshes.ParametrizedCurve}) = false
+supports_autoenzyme(::G) where {G <: Geometry} = supports_autoenzyme(G)
+
+_check_diff_method_support(::Geometry, ::DifferentiationMethod) = nothing
+function _check_diff_method_support(geometry::Geometry, ::AutoEnzyme)
+    if !supports_autoenzyme(geometry)
+        throw(ArgumentError("Differentiation method AutoEnzyme not supported for this geometry."))
```

Comment:
codecov reports this line is uncovered. Could you maybe add a test for that?

Reply:
I looked at codecov and it showed a different line? I tried to cover that, lets see how the
report comes out this time.

Reply:
Everything is covered now :+1: